Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -112,22 +112,36 @@ configuration:
   required: false
   type: 15
   section: Collect
-- additionalinfo: When selected, closing the Splunk notable event is mirrored in Cortex XSOAR.
+- additionalinfo: If selected, closing a Notable Event on Splunk with a status of "Closed" will close the corresponding fetched incident on XSOAR.
```

Comment:
If selected, closing a Notable Event in Splunk with a status of "Closed" will close the corresponding fetched incident in Cortex XSOAR.